config/prow/cluster/prow_controller_manager_deployment.yaml

```diff
@@ -22,6 +22,11 @@ metadata:
 spec:
   # Mutually exclusive with plank. Only one of them may have more than zero replicas.
   replicas: 1
+  strategy:
```

Comment:
would be nice to also add comment here as being done below. However not worth blocking